hooks/post/30-upgrade.rb

```diff
@@ -66,6 +66,15 @@ def remove_gutterball
   end
 end
 
+def remove_event_queue
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
```

Comment:
@chris1984 I believe @ekohl means that these paths are not change proof since the canonical source for the paths is contained within the puppet. If they change, this will break.